source/faq.txt

```diff
@@ -41,6 +41,59 @@ For applications that require a mix of the new and legacy APIs, ``com.mongodb.Mo
 - Configuration with ``MongoClientSettings`` and ``ConnectionString``, the only difference being that you create instances via constructors instead of a factory class.
 - CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``.  You can access this API via the ``getDatabase()`` method.
 
+.. _faq-legacy-connection:
+
+How do I connect to my MongoDB instance with the legacy API?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to connect to a MongoDB instance with the
+legacy API and the current API.
+
+Imagine we are connecting to a collection that contains only this document:
+
+.. code-block:: json
+   
+   {"_id": 1, "val": 1}
```

Comment:
Suggestion: I don't think an example is needed. Starting with showing your snippet below answers their question early on.

Reply:
I am not sure I agree with this suggestion. Here is my reasoning:

I think it is hard to understand code that does not produce output. If a sample document is not included, it becomes difficult to produce deterministic output from a code snippet (connection and server parameters and are different from deployment to deployment). Having code that produces output gives readers a natural way to test the code.